src/Microsoft.VisualBasic/tests/VB/FileIOTests.vb

```diff
@@ -0,0 +1,901 @@
+        End Function
+
+        <Fact>
+        Public Sub CombinePathTest_BadBaseDirectory_RelativePath()
+            Assert.Throws(Of ArgumentNullException)(Function() FileSystem.CombinePath("", "Test2"))
+            Assert.Equal(FileSystem.CombinePath("C:\", ""), "C:\")
+        End Sub
+
+        <Fact>
+        Public Sub CombinePathTest_BaseDirectory_RelativePath()
+            Assert.Equal(FileSystem.CombinePath("C:\", "Test2"), IO.Path.Combine("C:\", "Test2"))
+        End Sub
+        <Fact>
+        Public Sub CopyDirectory_SourceDirectoryName_DestinationDirectoryName()
+            Dim TestBase As New FileIOTestBase
+            Dim FullPathToSourceDirectory As String = IO.Path.Combine(TestBase.TestDirectory(), "SourceDirectory")
+            Dim FullPathToTargetDirectory As String = IO.Path.Combine(TestBase.TestDirectory(), "TargetDirectory")
+            IO.Directory.CreateDirectory(FullPathToSourceDirectory)
+            For i As Integer = 0 To 5
+                CreateTestFile(TestBase, SourceData, IO.Path.Combine(FullPathToSourceDirectory, $"NewFile{i}"))
+            Next
+            FileSystem.CopyDirectory(FullPathToSourceDirectory, FullPathToTargetDirectory)
+            Assert.Equal(IO.Directory.GetFiles(FullPathToSourceDirectory).Count, IO.Directory.GetFiles(FullPathToTargetDirectory).Count)
+            For Each CurrentFile As String In IO.Directory.GetFiles(FullPathToTargetDirectory)
+                ' Ensure copy transferred written data
+                Using stream As New IO.StreamReader(IO.File.OpenRead(CurrentFile))
+                    Dim ReadData(SourceData.Length - 1) As Char
+                    stream.Read(ReadData, 0, SourceData.Length)
+                    Assert.Equal(SourceData, ReadData)
+                End Using
+            Next
+            IO.Directory.Delete(FullPathToTargetDirectory, recursive:=True)
+            IO.Directory.CreateDirectory(FullPathToTargetDirectory)
+            CreateTestFile(TestBase, SourceData, IO.Path.Combine(FullPathToTargetDirectory, $"NewFile0"))
+            Assert.Throws(Of IO.IOException)(Sub() FileSystem.CopyDirectory(FullPathToSourceDirectory, FullPathToTargetDirectory))
+            TestBase.Dispose()
+        End Sub
+
+        <Fact>
+        Public Sub CopyDirectory_SourceDirectoryName_DestinationDirectoryName_OverwriteFalse()
+            Dim TestBase As New FileIOTestBase
+            Dim FullPathToSourceDirectory As String = IO.Path.Combine(TestBase.TestDirectory(), "SourceDirectory")
+            Dim FullPathToTargetDirectory As String = IO.Path.Combine(TestBase.TestDirectory(), "TargetDirectory")
+            IO.Directory.CreateDirectory(FullPathToSourceDirectory)
+            For i As Integer = 0 To 5
+                CreateTestFile(TestBase, SourceData, IO.Path.Combine(FullPathToSourceDirectory, $"NewFile{i}"))
+            Next
+            FileSystem.CopyDirectory(FullPathToSourceDirectory, FullPathToTargetDirectory, overwrite:=False)
```

Comment:
add spaces around ```overwrite := False```

Reply:
VS does allow it, 15.8.1 removes them automatically. If that needs to change the bug is in the VS formatter.